artifacts/AdditionalSetup.ps1

```diff
@@ -1,3 +1,65 @@
+function Move-Database {
+    param (
+        $databaseToMove
+    )
+
+    Write-Host " - Moving SaaS database to volume"
+    if ($env:volPath -ne "") {
+        $volPath = $env:volPath
+        [reflection.assembly]::LoadWithPartialName("Microsoft.SqlServer.Smo") | Out-Null
+        [reflection.assembly]::LoadWithPartialName("Microsoft.SqlServer.Common") | Out-Null
+        $dummy = new-object Microsoft.SqlServer.Management.SMO.Server
+        $sqlConn = new-object Microsoft.SqlServer.Management.Common.ServerConnection
+        $smo = new-object Microsoft.SqlServer.Management.SMO.Server($sqlConn)
+        $smo.Databases | Where-Object { $_.Name -eq $databaseToMove } | ForEach-Object {
+            # set recovery mode and shrink log
+            $sqlcmd = "ALTER DATABASE [$($_.Name)] SET RECOVERY SIMPLE WITH NO_WAIT"
+            & sqlcmd -Q $sqlcmd
```

Comment:
can you please set the ServerInstance to all sqlcmd's? Without it doesn't work on Windows Server 2019 (which we currently use in K8s): https://github.com/cosmoconsult/container-scripts-for-selfservice/commit/f45b95964d149c48e491e4b83e67d3b5221b8729

Reply:
@lippertmarkus like this? https://github.com/cosmoconsult/container-scripts-for-selfservice/pull/65/commits/97ea5ef285bcb88c95b5f4bc51aada71cacbef03